tests/metaworld/envs/mujoco/sawyer_xyz/test_scripted_policies.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from metaworld.envs.mujoco.env_dict import ALL_ENVIRONMENTS
+from metaworld.policies import *
+from tests.metaworld.envs.mujoco.sawyer_xyz.utils import check_success
+
+
+test_data = [
+    # name,      policy,      success rate,   environment kwargs
+    ['door-open-v1', SawyerDoorOpenV1Policy(), 0.95, {}],
+    ['door-close-v1', SawyerDoorCloseV1Policy(), 0.95, {}],
+    # Expected success is low because noise has huge negative impact on drawer-open
+    ['drawer-open-v1', SawyerDrawerOpenV1Policy(), 0.50, {}],
+    ['drawer-close-v1', SawyerDrawerCloseV1Policy(), 0.84, {}]
+]
+
+for row in test_data:
+    row[-1] = ALL_ENVIRONMENTS[row[0]](random_init=True, **row[-1])
+
+
+@pytest.mark.parametrize('label,policy,expected_success_rate,env', test_data)
+def test_scripted_policy(env, policy, label, expected_success_rate, iters=1000):
+    """Tests whether a given policy solves an environment in a stateless manner
+
+    Args:
+        env (metaworld.envs.MujocoEnv): Environment to test
+        policy (metaworld.policies.policy.Policy): Policy that's supposed to succeed in env
+        label (string): A string to use when printing out errors
+        expected_success_rate (float): Decimal value indicating % of runs that must be successful
+        iters (int): How many times the policy should be tested
+
+    """
+    assert len(vars(policy)) == 0, label + ' policy has state variable(s)'
+
+    successes = 0
+    for i in range(iters):
```

Comment:
please make this simpler calculating an overall success rate and then asserting that `success_rate >= expected_success_rate`